plexpy/__init__.py

```diff
@@ -547,13 +598,26 @@ def dbcheck():
             'ALTER TABLE session_history_metadata ADD COLUMN tagline TEXT'
         )
 
-    # notify_log table :: This is a table which logs notifications sent
-    c_db.execute(
-        'CREATE TABLE IF NOT EXISTS notify_log (id INTEGER PRIMARY KEY AUTOINCREMENT, '
-        'session_key INTEGER, rating_key INTEGER, user_id INTEGER, user TEXT, '
-        'agent_id INTEGER, agent_name TEXT, on_play INTEGER, on_stop INTEGER, on_watched INTEGER, '
-        'on_pause INTEGER, on_resume INTEGER, on_buffer INTEGER)'
-    )
+    # Upgrade session_history_metadata table from earlier versions
+    try:
+        c_db.execute('SELECT library_id from session_history_metadata')
+    except sqlite3.OperationalError:
+        logger.debug(u"Altering database. Updating database table session_history_metadata.")
+        c_db.execute(
+            'ALTER TABLE session_history_metadata ADD COLUMN library_id INTEGER'
+        )
+        c_db.execute(
+            'ALTER TABLE session_history_metadata ADD COLUMN library_title TEXT'
+        )
+        logger.debug(u"Updating library_id's in database. Please wait...")
+        from plexpy import datafactory
+        data_factory = datafactory.DataFactory()
+        result = data_factory.update_library_ids()
```

Comment:
Maybe make sure the server is online and that we are connected to the correct server before running this.

This should preferably be invoked manually by the user.